.github/workflows/build.yml

```diff
@@ -33,5 +26,131 @@ jobs:
         uses: actions/upload-artifact@v3
         if: success()
         with:
-          name: artifacts-${{ matrix.kernel_version }}-${{ matrix.libc }}
+          name: artifacts-5.15.19-glibc
           path: artifacts
+
+  test:
+    name: Test
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: "debian11", url: "https://cache.netdata.cloud/vagrant/debian/debian11/vm.json"}
+          - {name: "ubuntu2110", url: "https://cache.netdata.cloud/vagrant/ubuntu/ubuntu2110/vm.json"}
+          - {name: "ubuntu2204", url: "https://cache.netdata.cloud/vagrant/ubuntu/ubuntu2204/vm.json"}
+          - {name: "rockylinux8", url: "https://cache.netdata.cloud/vagrant/rockylinux/rockylinux8/vm.json"}
+          - {name: "almalinux8", url: "https://cache.netdata.cloud/vagrant/almalinux/almalinux8/vm.json"}
+          - {name: "almalinux9", url: "https://cache.netdata.cloud/vagrant/almalinux/almalinux9/vm.json"}
+          - {name: "fedora35", url: "https://cache.netdata.cloud/vagrant/fedora/fedora35/vm.json"}
+          - {name: "fedora36", url: "https://cache.netdata.cloud/vagrant/fedora/fedora36/vm.json"}
+          - {name: "oraclelinux8", url: "https://cache.netdata.cloud/vagrant/oraclelinux/oraclelinux8/vm.json"}
+          - {name: "alpine313", url: "https://cache.netdata.cloud/vagrant/alpine/alpine313/vm.json"}
+          - {name: "alpine314", url: "https://cache.netdata.cloud/vagrant/alpine/alpine314/vm.json"}
+          - {name: "alpine315", url: "https://cache.netdata.cloud/vagrant/alpine/alpine315/vm.json"}
+          - {name: "alpine316", url: "https://cache.netdata.cloud/vagrant/alpine/alpine316/vm.json"}
+          - {name: "archlinux", url: "https://cache.netdata.cloud/vagrant/archlinux/archlinux/vm.json"}
+          - {name: "opensuse153", url: "https://cache.netdata.cloud/vagrant/opensuse/opensuse153/vm.json"}
+          - {name: "opensuse154", url: "https://cache.netdata.cloud/vagrant/opensuse/opensuse154/vm.json"}
+    runs-on: macos-10.15
+    needs: build-artifacts
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Retrieve Dist Tarball
+        id: fetch-dist
+        uses: actions/download-artifact@v3
+        with:
+          name: artifacts-5.15.19-glibc
+          path: src
+      - name: Extract artifact
+        working-directory: src
+        run: |
+          tar -xzf netdata_ebpf-co-re-glibc.tar.xz
+      - name: Update vagrant and virtualbox
+        working-directory: src
+        run: |
+          brew upgrade vagrant
+          brew upgrade --cask virtualbox
```

Comment:
Do we really need to use `brew` on Linux hosts?

Reply:
the Github workers are Mac.
The tasks are running in VirtualBox VMs of various kind.